apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationComment.jsx

```diff
@@ -2,58 +2,58 @@ import React, { Component } from 'react'
 import django from 'django'
 
 export default class ModerationComment extends Component {
-  constructor (props) {
-    super(props)
-
-    this.state = {
-      project: this.props.key
-    }
-  }
-
   render () {
+    const { classification, commentText, commentUrl, created, userImage, userName, aiClassified } = this.props
     const postedText = django.gettext(' posted a ')
-    const offensiveText = django.gettext(' for being offensive')
-    const categoryText = django.gettext('Categories: ')
+    const offensiveTextReport = django.gettext(' that has been reported as ')
```

Comment:
Uh, I guess, because of the translations rebasing might stop here. You just need to use 
`django.pgettext('kosmo', ' that has been...` for all the strings, then.